src/tsolvers/lasolver/LASolver.h

```diff
@@ -24,6 +24,17 @@
 #include <unordered_set>
 
 namespace opensmt {
+class LANonLinearException : public std::runtime_error {
+public:
+    LANonLinearException(char const * reason_) : runtime_error(reason_) {
```

Comment:
Let's modernize this and use `std::string_view`.

Reply:
Updated constructor to receive `std::string_view`